### PR TITLE
[CDAP-12898] add Reports Customizer section

### PR DESCRIPTION
**Pull request by elfenheart**
JIRA: https://issues.cask.co/browse/CDAP-12898

### CODE REVIEW

File: cdap-ui/app/cdap/components/Reports/Customizer/index.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  render() {
+    return (
+      <div className={classnames('customizer-container', {collapsed: this.state.isCollapsed})}>
+        <div className="collapsed-toggle-container">
+          <div
+            className="toggle"
+            onClick={this.toggleCollapsed}
+          >
+            <IconSVG name={this.state.isCollapsed ? 'icon-caret-right' : 'icon-caret-down'} />
+            {this.state.isColapsed ? 'Show generate a report' : 'Hide generate a report'}
```
Comment (tonybach) [inline cdap-ui/app/cdap/components/Reports/Customizer/index.js]: typo: should be `isCollapsed`

File: cdap-ui/app/cdap/components/Reports/store/ReportsStore.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+ */
+
+import {combineReducers, createStore} from 'redux';
+
+const ReportsActions = {
+  toggleCustomizerOption: 'REPORTS_TOGGLE_CUSTOMIZER_OPTION',
+  reset: 'REPORTS_RESET'
+};
+
+const defaultAction = {
```
Comment (tonybach) [inline cdap-ui/app/cdap/components/Reports/store/ReportsStore.js]: nit: Can get `defaultAction` from `services/helpers`

**Review by tonybach — COMMENTED** [review]
Two minor comments, rest of code LGTM 👍

File: cdap-ui/app/cdap/components/Reports/Customizer/index.js (hunk 1 above)
Comment (elfenheart) [reply cdap-ui/app/cdap/components/Reports/Customizer/index.js]: nice catch 👍

**Comment by tonybach** [comment]
Code LGTM 👍

**Pull request merged**